### PR TITLE
Changes to support installing the devkit on linux/s390x build images

### PR DESCRIPTION
**Pull request by sxa**
Just noticed that this has not yet been merged - the jobs that use it are currently configured to run from this branch.

##### Checklist

- [ ] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [x] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [x] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access) https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/1885/
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -85,16 +85,30 @@ have at the moment:
...
 | [Centos6](./ansible/docker/Dockerfile.CentOS6) | [`adoptopenjdk/centos6_build_image`](https://hub.docker.com/r/adoptopenjdk/centos6_build_image)| linux/amd64 | [GH Actions](.github/workflows/build.yml) | Yes
 | [Alpine3](./ansible/docker/Dockerfile.Alpine3) | [`adoptopenjdk/alpine3_build_image`](https://hub.docker.com/r/adoptopenjdk/alpine3_build_image) | linux/x64 & linux/arm64 | [Jenkins](https://ci.adoptium.net/job/centos7_docker_image_updater/) | Yes
 | [Ubuntu 20.04 (riscv64 only)](./ansible/docker/Dockerfile.Ubuntu2004-riscv64) | [`adoptopenjdk/ubuntu2004_build_image:linux-riscv64`](https://hub.docker.com/r/adoptopenjdk/ubuntu2004_build_image) | linux/riscv64 | [Jenkins](https://ci.adoptium.net/job/centos7_docker_image_updater/) | Yes
 
+<details>
+<summary>(*) - Caveats:</summary>
+
+The RHEL7 image creation for s390x has to be run on a RHEL host using a
+container implementation supplied by Red Hat, and we are using RHEL8 for
+this as it has a stable implemention.  The image creation requires the
```
Comment (karianna) [inline FAQ.md:98]:
```suggestion
this as it has a stable implemention. The image creation requires the
```

File: FAQ.md [hunk 2]
```diff
@@ -85,16 +85,30 @@ have at the moment:
...
+following:
+
+1. The host needs to have an active RHEL subscription
+2. The RHEL7 devkit (which cannot be made public) to be available in a tar file under /usr/local on the host as per the name in the Dockerfile
```
Comment (karianna) [inline FAQ.md:102]:
```suggestion
2. The RHEL7 devkit (which cannot be made public) is to be available in a tar file under `/usr/local` on the host as per the name in the Dockerfile.
```

File: FAQ.md [hunk 3]
```diff
@@ -85,16 +85,30 @@ have at the moment:
...
 
+<details>
+<summary>(*) - Caveats:</summary>
+
+The RHEL7 image creation for s390x has to be run on a RHEL host using a
+container implementation supplied by Red Hat, and we are using RHEL8 for
+this as it has a stable implemention.  The image creation requires the
+following:
+
+1. The host needs to have an active RHEL subscription
```
Comment (karianna) [inline FAQ.md:101]:
```suggestion
1. The host needs to have an active RHEL subscription.
```

File: FAQ.md [hunk 4]
```diff
@@ -85,16 +85,30 @@ have at the moment:
...
 the appropriate CI system listed in the table above by configuring them with
-the ansible playbooks and pushing them up to Docker Hub where they can be
-consumed by our jenkins build agents when the `DOCKER_IMAGE` value is
-defined on the jenkins build pipelines as configured in the
-[pipeline_config files](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations).
+the ansible playbooks and - with the exception of the RHEL7 image for s390x -
+pushing them up to Docker Hub where they can be consumed by our jenkins
+build agents when the `DOCKER_IMAGE` value is defined on the jenkins build
+pipelines as configured in the [pipeline_config
+files](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations).
```
Comment (karianna) [inline FAQ.md:111]: Should that be Adoptium?

**Review by karianna — CHANGES_REQUESTED** [review]

File: FAQ.md [hunk 5]
```diff
@@ -85,16 +85,30 @@ have at the moment:
...
```
Comment (sxa) [inline FAQ.md:111]:
```suggestion
files](https://github.com/adoptium/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations).
```

File: FAQ.md (hunk 4 above)
Comment (sxa) [reply FAQ.md:111]: Yes - good spot. It redirects correctly but it should be correct in here. I'll fix

**Review by karianna — APPROVED** [review]

**Comment by sxa** [comment]
Dupliate of https://github.com/adoptium/infrastructure/pull/3492 - closing

**Pull request closed without merging**